sklearn/tree/export.py

```diff
@@ -152,7 +152,7 @@ def get_color(value):
             # Classification tree
             color = list(colors['rgb'][np.argmax(value)])
             sorted_values = sorted(value, reverse=True)
-            alpha = int(np.round(255 * (sorted_values[0] - sorted_values[1]) /
+            alpha = 0 if len(sorted_values) == 1 else int(np.round(255 * (sorted_values[0] - sorted_values[1]) /
                                  (1 - sorted_values[1]), 0))
```

Comment:
This is too long and not readable. Just use a regular multiline if / else instead of a ternary expression.